code/modules/mob/living/clothing/clothing.dm

```diff
@@ -0,0 +1,279 @@
+
+////////////////////////////////////////////////////////
+// Without host:
+// 	Vent crawl
+//		Nuff said.
+//
+// 	Able to leap onto a human, tear off their clothes. This also forces a transform the first time you do this to a host.
+//
+//  Someone can also willingly put you on, but where's the fun in that?
+//
+//	Able to open doors if you have an id in pocket
+//
+//  Can change look to any jumpsuits and a bunch of costumes. (or maybe you have to eat the clothes to change to them)
+//
+//  Doesn't need oxygen.
+//
+//  Resists crushing damage, vulnerable to slicing and fire damage.
+
+
+////////////////////////////////////////////////////////
+// Host 'benefits'
+//
+//	Blood regeneration
+//		You regenerate blood 3 times as fast.
+//
+//  Natural healing
+//		Wearing living clothing SLOWLY heals you from brute
+//
+//	Can no longer wear clothes over your living clothing.
+//
+//  Can't take off your clothes, have to ask the clothing to remove itself. (?)
+//
+//  Can strip people twice as fast.
+
+////////////////////////////////////////////////////////
+// With host (untransformed):
+//
+// 	Can change look as above.
+//
+//	Unconscious movement
+// 		If host goes unconscious, you can slowly move the host to somewhere safe.
+//
+// 	Transformation
+//		Drink some blood - you transform into a stronger form.
+//
+//  Untransform
+//		Can do this any time you are transformed, or if you take too much damage.
+//
+// 	Overload\Blood heat
+//		You absorb most of the blood of your host, transform into an extremely powerful form.
+//		The host goes unconscious and loses control. You now control movement.
+//		You lose some control too. You autoattack everything around you.
+//
+//  Eat clothing
+//		Lets you eat clothes. Heals you. Unworn shit works kind of crappily - the more worn the clothing the better.
+
+////////////////////////////////////////////////////////
+// With host (transformed)
+//
+//  All damage to host except for targeted at head is damage reduced and applied to you.
+//		Spews blood everywhere because you drank it from your host.
+//
+// 	Chem injections instead go to you. Probably have no effect?
+//
+//
+//	All forms have an initial blood cost for the transformation and slowly drain blood
+//
+// 	Flight form
+//		Lets you jetpack around, even with gravity on
+//
+// 	Blades form
+//		Gain armblade (steal from ling, gogogo), people get damage if they attack\grab you.
+//		Transforming breaks cuffs.
+//
+
+////////////////////////////////////////////////////////
+// the mob code
+
+/mob/living/clothing
+	languages_spoken = HUMAN // TODO: replace with CLOTHING. People you possess gain the ability to speak CLOTHING
+	languages_understood = HUMAN
+
+	pass_flags = PASSTABLE | PASSMOB
+	mob_size = MOB_SIZE_SMALL
+
+	ventcrawler = 2
+
+	blood_volume = 0 // We want this?
+
+	// Damage vars
+	var/melee_damage_lower = 1
+	var/melee_damage_upper = 1
+	var/obj_damage = 0 //how much damage this simple animal does to objects, if any
```

Comment:
Because then I have to make a bunch of lines disabling the stuff from simple_animal I don't want. I might change my stance on this later, though. If I do, it's not hard to change later.